perllib/FixMyStreet/DB/Result/Problem.pm

```diff
@@ -737,7 +744,7 @@ sub nearest_address {
 }
 
 sub body {
-    my ( $problem, $link ) = @_;
+    my ( $problem, $link, $successful_send_only ) = @_;
```

Comment:
Sorry for not mentioning this before, but only noticed as part of checking for this review - I don't think you need this additional argument, because it's never called with it not set? Worth a comment above the `body` function explaining it returns a string of the successfully sent body names, but I don't see a need to provide a variable that's not used in more than one way.